contribution/Killercoda-Kubearmor/kubearmor-installation-view.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+namespace="kubearmor"
+duration=120
+watch -n 1 "kubectl get po -n $namespace" &
+watch_pid=$!
+sleep $duration
+kubectl wait --for=condition=ready --timeout=$duration -n kubearmor pod -l kubearmor-app
+kill $watch_pid
```

Comment:
not required

Reply:
It's necessary to monitor the pods under the 'kubearmor' namespace until all of them are in a 'Running' condition to make it visually appealing. The 'watch_pid' is required to terminate it when all the pods are in a 'Running' condition. This improvement enhances the user experience for Killercoda.

Reply:
The above kubectl wait command will handle that case. It will only be successful until all the pods are in running state.
You can addd `kubectl get pods -A` after `kubectl wait` for showing the state.

Reply:
You can add sleep before showing the running deployments in the above command too. But it's alright for now ig

Reply:
kubectl wait command will suddenly exit when it detects that all pods are in the 'Running' state, which is great. However, the issue is that if one pod is in the 'Running' state while others haven't even started creating or shown, the command exits prematurely. It doesn't wait because it assumes there's only one pod in the 'Running' state. I hope this clarifies the issue."

Reply:
Okay, if that's the case then it's fine 👍